packages/x-tree-view/src/SimpleTreeView/SimpleTreeView.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { styled, useThemeProps } from '@mui/material/styles';
+import composeClasses from '@mui/utils/composeClasses';
+import { useSlotProps } from '@mui/base/utils';
+import { getSimpleTreeViewUtilityClass } from './simpleTreeViewClasses';
+import { SimpleTreeViewProps } from './SimpleTreeView.types';
+import { useTreeView } from '../internals/useTreeView';
+import { TreeViewProvider } from '../internals/TreeViewProvider';
+import { SIMPLE_TREE_VIEW_PLUGINS } from './SimpleTreeView.plugins';
+import { buildWarning } from '../internals/utils/warning';
+
+const useUtilityClasses = <Multiple extends boolean | undefined>(
+  ownerState: SimpleTreeViewProps<Multiple>,
+) => {
+  const { classes } = ownerState;
+
+  const slots = {
+    root: ['root'],
+  };
+
+  return composeClasses(slots, getSimpleTreeViewUtilityClass, classes);
+};
+
+export const SimpleTreeViewRoot = styled('ul', {
+  name: 'MuiSimpleTreeView',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.root,
+})<{ ownerState: SimpleTreeViewProps<any> }>({
+  padding: 0,
+  margin: 0,
+  listStyle: 'none',
+  outline: 0,
+});
+
+type SimpleTreeViewComponent = (<Multiple extends boolean | undefined = undefined>(
+  props: SimpleTreeViewProps<Multiple> & React.RefAttributes<HTMLUListElement>,
+) => React.JSX.Element) & { propTypes?: any };
+
+const EMPTY_ITEMS: any[] = [];
+
+const itemsPropWarning = buildWarning([
+  'The `SimpleTreeView` component does not support the `items` prop.',
+  'If you want to add items, you need to pass them as JSX children',
+  'Check you the documentation for more details: https://mui.com/x/react-tree-view/items/#usage-with-treeview',
```

Comment:
Does such a page still exist? 🤔 
```suggestion
  'Check the documentation for more details: https://mui.com/x/react-tree-view/simple-tree-view/items/',
```

Reply:
I did not change these links when I updated the doc... thanks for the cast.